python_modules/libraries/dagster-aws/dagster_aws_tests/pipes_tests/test_pipes.py

```diff
@@ -293,12 +295,18 @@ def script_fn():
             open_dagster_pipes,
         )
 
-        client = boto3.client("s3", region_name="us-east-1", endpoint_url="http://localhost:5193")
-        context_loader = PipesS3ContextLoader(client=client)
-        message_writer = PipesS3MessageWriter(client, interval=0.001)
+        s3_client = boto3.client(
+            "s3", region_name="us-east-1", endpoint_url="http://localhost:5193"
+        )
+
+        messages_backend = os.environ["PIPES_MESSAGES_BACKEND"]
```

Comment:
is this an existing env var? only in test?

Reply:
This is only for testing (to parametrize over message passing options). I will rename it to `TESTING_PIPES_MESSAGES_BACKEND` to remove confusion.